From: Shivank Garg <shivankg@amd.com>
Subject: jfs: fix kernel-doc warning in jfs_metapage.c
Date: Thu, 1 May 2025 11:10:54 +0000

Change comment style from "/**" to "/*" as the metapage_migrate_folio
function comment is not kernel-doc.

Link: https://lkml.kernel.org/r/1967593d-8084-4a4a-b384-35d5adc54eb4@amd.com
Signed-off-by: Shivank Garg <shivankg@amd.com>
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202505011647.L0LGO7Lm-lkp@intel.com/
Cc: Alistair Popple <apopple@nvidia.com>
Cc: Dan Carpenter <dan.carpenter@linaro.org>
Cc: Dave Kleikamp <shaggy@kernel.org>
Cc: David Hildenbrand <david@redhat.com>
Cc: Donet Tom <donettom@linux.ibm.com>
Cc: Jane Chu <jane.chu@oracle.com>
Cc: Kefeng Wang <wangkefeng.wang@huawei.com>
Cc: Matthew Wilcox <willy@infradead.org>
Cc: Zi Yan <ziy@nvidia.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 fs/jfs/jfs_metapage.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/fs/jfs/jfs_metapage.c~jfs-implement-migrate_folio-for-jfs_metapage_aops-fix
+++ a/fs/jfs/jfs_metapage.c
@@ -629,7 +629,7 @@ static bool metapage_release_folio(struc
 	return ret;
 }
 
-/**
+/*
  * metapage_migrate_folio - Migration function for JFS metapages
  */
 static int metapage_migrate_folio(struct address_space *mapping, struct folio *dst,
_
